packages/xrpl/test/utils/deriveXAddress.ts

```diff
@@ -22,4 +22,17 @@ describe('deriveXAddress', function () {
       'TVVrSWtmQQssgVcmoMBcFQZKKf56QscyWLKnUyiuZW8ALU4',
     )
   })
+
+  it('does not include tag when null', function () {
+    assert.equal(
+      deriveXAddress({
+        publicKey:
+          'ED02C98225BD1C79E9A4F95C6978026D300AFB7CA2A34358920BCFBCEBE6AFCD6A',
+        // @ts-expect-error -- Assessing null behavior
+        tag: null,
```

Comment:
```suggestion
        tag: undefined,
```
This might help with the TS error

Reply:
Unfortunately it did not, but because we're now trying to have `undefined` match as well I added a test case for it :)